CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+- Renamed `VERSION` file to `CHANGELOG.md`
```

Comment:
Hey @joes-myob , hubr looks for a `VERSION` file and uses that for releasing and bumping, if you rename it to CHANGELOG and change how it's organized some of hubr's functionality will break.

Reply:
ah ha - thanks for pointing that out - can you link me to the code where it's looking for the VERSION file?

Reply:
Looks like there is an arg to the path of the VERSION file
https://github.com/myob-oss/hubr/blob/master/main.go#L1389 - but I still cant see _where_ the code is looking for a "VERSION" file? @PhonesMYOB

Reply:
I reverted these changes just in case

Reply:
https://github.com/myob-oss/hubr/blob/master/main.go#L1394
you'll also need to update comments and docs that refer to VERSION